production/helm/loki/values.yaml

```diff
@@ -3292,9 +3292,8 @@ resultsCache:
   #  whenUnsatisfiable: ScheduleAnyway
   # -- Tolerations for results-cache pods
   tolerations: []
-  # -- Pod Disruption Budget
-  podDisruptionBudget:
-    maxUnavailable: 1
+  # -- Pod Disruption Budget maxUnavailable
+  maxUnavailable: 1
```

Comment:
for record: this feels like an breaking change, but `podDisruptionBudget.maxUnavailable` was not used or referenced in the templates.

Reply:
Yeah, I thought the same and that's why I changed it to the style the other `maxUnavailable` variables have for the other services.